tests/Rector/Composer/ExtensionComposerRector/Fixture/update_package_no_extra.json

```diff
@@ -9,15 +9,15 @@
 -----
 {
     "name": "foo/bar",
-    "license": "GPL-2.0-or-later",
     "description": "Add description...",
+    "license": "GPL-2.0-or-later",
+    "type": "typo3-cms-extension",
+    "require": {
+        "typo3/cms-core": "^9.5"
+    },
     "extra": {
         "typo3/cms": {
             "extension-key": "_temp_fixture_easy_testing"
         }
-    },
-    "type": "typo3-cms-extension",
-    "require": {
-        "typo3/cms-core": "^9.5"
     }
```

Comment:
Rector ComposerJson printer is using more standardized order of keys. E.g `require` is in the top.